.env.local

```diff
@@ -0,0 +1,30 @@
+CELERY_TASK_ALWAYS_EAGER=True
+CELERY_TASK_EAGER_PROPAGATES=True
```

Comment:
CELERY_TASK_ALWAYS_EAGER and CELERY_TASK_EAGER_PROPAGATES are not loaded from environment variables in settings, remove them here